runtime/src/main/java/com/flipkart/flux/guice/module/ConfigModule.java

```diff
@@ -53,23 +65,42 @@ protected void configure() {
      * @Named("Dashboard.service.port") int port
      */
     @SuppressWarnings({ "rawtypes", "unchecked" })
-    private void bindConfigProperties(URL url) {
-        try {
-            YamlConfiguration yamlConfiguration = new YamlConfiguration(url);
-            bind(YamlConfiguration.class).toInstance(yamlConfiguration);
-            Iterator<String> propertyKeys = yamlConfiguration.getKeys();
-            while (propertyKeys.hasNext()) {
-                String propertyKey = propertyKeys.next();
-                Object propertyValue = yamlConfiguration.getProperty(propertyKey);
-                LinkedBindingBuilder annotatedWith = bind(propertyValue.getClass()).annotatedWith(Names.named(propertyKey));
-                annotatedWith.toInstance(propertyValue);
-            }
-        } catch (IOException e) {
-            throw new RuntimeException(e);
+    private void bindConfigProperties() {
```

Comment:
We were using the url to construct YamlConfiguration, as I've put it as a class field to use in multiple methods, we don't need to pass this param.